ludwig/backend/ray.py

```diff
@@ -623,10 +623,10 @@ def __init__(
         self.df_engine = df_engine
 
     def get_trainer_kwargs(self) -> Dict[str, Any]:
-        return {**self.trainer_kwargs, **get_trainer_kwargs()}
+        return {**get_trainer_kwargs(), **self.trainer_kwargs}
 
     def get_resources_per_worker(self) -> Tuple[int, int]:
-        trainer_kwargs = {**self.trainer_kwargs, **get_trainer_kwargs()}
+        trainer_kwargs = {**get_trainer_kwargs(), **self.trainer_kwargs}
```

Comment:
@magdyksaleh Should this line just be a call to `self.get_trainer_kwargs()` and we just make this change/fix within `get_trainer_kwargs` function defined for `RayPredictor`? I think we don't actually need to do this twice and this was a typo/oversight from past work on the `RayPredictor` class.

Reply:
agree that's cleaner